### PR TITLE
Remove deprecated "X-" in headers

### PR DESCRIPTION
**Pull request by harianus**
The X in headers is deprecated http://tools.ietf.org/html/rfc6648.

I don't know how the `X-Prerender-Token` works, so I kept it in.

```erb
<% if ENV["PRERENDER_TOKEN"] %>
    proxy_set_header X-Prerender-Token <%= ENV["PRERENDER_TOKEN"] %>;
<% end %>
```

### CODE REVIEW

**Comment by tonycoco** [comment]
I'm not sure this is the case.

Almost all of those X headers are useful/standardized on an Nginx setup. Also, the Prerender.io headers are needed. The other headers that are for debugging I use to check with a cURL script.

File: config/nginx.conf.erb [hunk 1]
```diff
@@ -52,12 +52,12 @@ http {
     <% if ENV["API_URL"] %>
       location <%= ENV["API_PREFIX_PATH"] || "/api/" %> {
         proxy_pass <%= ENV["API_URL"] %>;
-        proxy_set_header X-Real-IP $remote_addr;
```
Comment (tonycoco) [inline config/nginx.conf.erb]: http://nginx.org/en/docs/http/ngx_http_proxy_module.html#example -- this is straight from the Nginx docs.

**Comment by harianus** [comment]
For Prerender.io I'm not sure. They commented the line in their official script out: https://gist.github.com/thoop/8165802

But the RFC is a SHOULD NOT, so it is not forbidden ;-)

**Comment by tonycoco** [comment]
Could you rebase this @harianus? I'll merge once that is done. Sorry I forgot to get to this!

**Comment by harianus** [comment]
Done, sorry for the empty commit https://github.com/harianus/heroku-buildpack-ember-cli/commit/eba0e93

**Pull request merged**